studio/components/interfaces/Settings/Database/NetworkRestrictions/AddRestrictionModal.tsx

```diff
@@ -34,7 +34,9 @@ const AddRestrictionModal = ({
 
     // Validate CIDR block size
     const isOutOfCidrSizeRange = cidrBlockSize < 0 || cidrBlockSize > 32
-    if (isOutOfCidrSizeRange) errors.cidrBlockSize = 'Size has to be between 0 to 32'
+    if (cidrBlockSize.length === 0 || isOutOfCidrSizeRange) {
```

Comment:
I think 0 may be valid value actually – maybe check with infra?

Reply:
yeap 0 is a valid value - the additional check here just verifies if the input field is empty

Reply:
so validation will say "no" if input field is empty, or a number below 0 or higher than 32 is given

Reply:
ah my bad! nice